neocore/Cryptography/Helper.py

```diff
@@ -88,7 +71,10 @@ def scripthash_to_address(scripthash):
     Returns:
         str: base58 encoded string representing the wallet address.
     """
-    return bin_to_b58check(binascii.unhexlify(scripthash), 23)
+    sb = bytearray([23]) + scripthash
```

Comment:
would love to get the thoughts of @localhuman on this refactoring. looks good to me tho.